db.go

```diff
@@ -1146,7 +1146,7 @@ func (db *DB) NewIterator(opts *ReadOptions) *Iterator {
 // that uses the ReadOptions given.
 func (db *DB) NewIteratorCF(opts *ReadOptions, cf *ColumnFamilyHandle) *Iterator {
 	cIter := C.rocksdb_create_iterator_cf(db.c, opts.c, cf.c)
-	return newNativeIterator(cIter)
+	return newNativeIteratorWithTS(cIter, opts.timestamp)
```

Comment:
there are a few other bytes fields in `ReadOptions`, need to handle in a similar way?

Reply:
yes, if it's ok with this direction